package.json

```diff
@@ -7,7 +7,8 @@
     "tap": "tap",
     "test": "tap test/*.js --100",
     "coverage": "tap test/*.js --coverage-report=html",
-    "updatesnapshots": "TAP_SNAPSHOT=1 npm run test"
+    "updatesnapshots": "TAP_SNAPSHOT=1 npm run test",
+    "ts-node": "ts-node"
```

Comment:
Can this and the extra devDependencies live in an extra package.json within the typescript examples directory?

Reply:
I created a project in the `examples/typescript` folder so I am treating that as a separate project. Can you please tell me if it's a suitable solution?

Reply:
Yeah, that's perfect!